core_impl/src/test/java/io/opencensus/stats/MutableViewDataTest.java

```diff
@@ -15,19 +15,108 @@
 
 import static com.google.common.truth.Truth.assertThat;
 
+import com.google.common.collect.ImmutableMap;
+import io.opencensus.stats.Aggregation.Count;
+import io.opencensus.stats.Aggregation.Histogram;
+import io.opencensus.stats.Aggregation.Mean;
+import io.opencensus.stats.Aggregation.Range;
+import io.opencensus.stats.Aggregation.StdDev;
+import io.opencensus.stats.Aggregation.Sum;
+import io.opencensus.stats.AggregationData.CountData;
+import io.opencensus.stats.AggregationData.HistogramData;
+import io.opencensus.stats.AggregationData.MeanData;
+import io.opencensus.stats.AggregationData.RangeData;
+import io.opencensus.stats.AggregationData.StdDevData;
+import io.opencensus.stats.AggregationData.SumData;
+import io.opencensus.stats.MutableAggregation.MutableCount;
+import io.opencensus.stats.MutableAggregation.MutableHistogram;
+import io.opencensus.stats.MutableAggregation.MutableMean;
+import io.opencensus.stats.MutableAggregation.MutableRange;
+import io.opencensus.stats.MutableAggregation.MutableStdDev;
+import io.opencensus.stats.MutableAggregation.MutableSum;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
 
-/**
- * Tests for {@link MutableViewData}.
- */
+/** Tests for {@link MutableViewData}. */
 @RunWith(JUnit4.class)
 public class MutableViewDataTest {
 
+  private static final double EPSILON = 1e-7;
+
   @Test
   public void testConstants() {
     assertThat(MutableViewData.UNKNOWN_TAG_VALUE.asString()).isEqualTo("unknown/not set");
   }
 
+  @Test
+  public void testGetTagValues() {
+    TagKey originator = TagKey.create("originator");
```

Comment:
We generally want to encourage people to make TagKeys (and TagValues, when possible) to be static constants so it's worth doing here too.